sass/_typography.scss

```diff
@@ -165,6 +165,7 @@ h6,
     width: fit-content;
     line-height: 150%;
     position: relative;
+    padding: 0 !important;
```

Comment:
But why `!important`? You should use it only, if nothing else helps.

Reply:
When you use it f.e. on a heading, the padding of the heading overwrites it.